docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Ubuntu/README.md

```diff
@@ -1 +1,165 @@
-Ubuntu specific instructions
\ No newline at end of file
+# Installing LORIS In-Depth
+
+This guide assumes an installation on an Ubuntu server. 
+
+If you are using CentOS, please visit the CentOS installation guide for 
+instructions on installation.
+
+We also have instructions for how to set up LORIS for macOS. However, only
+ Ubuntu and CentOS are officially supported by the team so we may be unable to
+ help you with issues arising on other systems.
+
+When you've completed this guide, you should be able to load and log into LORIS in your browser. Further setup to customize LORIS for your project will be required after that. Please visit
+[the Setup page](https://github.com/aces/Loris/wiki/Setup) in order to complete the setup for LORIS.
+
+## Prequisities
+
+### Server Stack
+
+LORIS requires a LAMP stack in order to run, specifically:
+* Apache 2.4
+* MySQL 5.7 (or MariaDB 10.3) (or higher)
+* PHP 7.3 (or higher)
+
+Additionally, LORIS requires the following package managers:
+* npm 8.0 (or higher)
+* composer
+
+This guide does not cover installation of these requirements.
+
+### Apt Packages
+The following Ubuntu packages are required and should be installed using 
+`sudo apt install ...`.
+
+* git
+* curl
+* wget
+* zip
+* unzip
+* php-json
+* make
+* software-properties-common
+* php7.3-mysql
+* php7.3-xml
+* php7.3-mbstring
+* php7.3-gd
+* libapache2-mod-php7.3
+
+## Getting the source code
+
+### LORIS for Development
+If installing LORIS for development purposes, clone the source code from our
+GitHub repository to the web root. Then change to this directory.
+
+Example command assuming LORIS will be served from `/var/www/`:
+
+```bash
+git clone git@github.com:aces/Loris.git /var/www/loris
+cd /var/www/loris/
+```
+
+### LORIS for Production
+
+Visit the [releases](github.com/aces/loris/releases) page and download the zipped file of the latest LORIS
+release. The below command will download the latest LORIS release and save 
+it to a file named `loris-src.tar.gz`.
+
+```bash
+curl -s https://api.github.com/repos/aces/loris/releases/latest \
+| grep 'tarball' \
+| cut -d : -f2,3 \
+| tr -d \" \
+| tr -d , \
+| wget -i - -O loris-src.tar.gz
+```
+
+When this is complete, expand the compressed file and move its contents to the
+directory from which LORIS will be served. 
+
+```bash
+tar -zxf loris-src.tar.gz
+```
+
+This will create a folder called something like `aces-Loris-9e30cf0`. (The
+part after `aces-Loris` is not important).
+
+Rename this file to `loris`, e.g.
+
+```bash
+mv aces-Loris-9e30cf0 loris
+```
+
+Then move this folder to the web root (e.g. `/var/www/`) and go to this 
+directory.
+
+```bash
+mv ./loris /var/www/; cd /var/www/loris
+
+### If installing LORIS for production
+
+```bash
+cd `/var/www/loris/`; make
+```
+
+### If installing LORIS for development
```

Comment:
I think it's confusing that this section is here and not in the "Development" section below.. following these instructions will also result in a "Development" environment that was installed from a tarball and isn't tracked by git. I think this document should simply have an audience and stick to it rather than trying to provide different details to different audiences.

Reply:
Agreed I'll refactor it